src/Scene/Node.js

```diff
@@ -90,14 +90,17 @@ Node.prototype.update = function() {
  * @param {type} level
  * @returns {undefined}
  */
-Node.prototype.getParentLevel = function(level) {
+Node.prototype.getNodeAtLevel = function(level) {
```

Comment:
I changed the name because I also changed the behavior.
If `this.level == level` then it returns `this` - so it's not always the parent